packages/snowpack/src/config.ts

```diff
@@ -378,6 +374,9 @@ function loadPlugins(
         const result = await plugin.build({
           ...options,
           contents: fs.readFileSync(options.filePath, 'utf-8'),
+        }).catch((err) => {
+          console.error(`[${plugin.name}] ERROR: There was a problem running this older plugin. Please update the plugin to the latest version.`);
```

Comment:
not expected, but added here just in case an error does occur so that the user knows how to resolve